### DISCUSSION TITLE
Zoom recording options

### DISCUSSION THREAD

**Discussion by danieltott** [Virtual-Coffee/VC-Community-Docs#157] [Audio/Visual]
When recording zoom calls, Zoom gives us some options and I wanted to check in with @BogDAAAMN and the @Virtual-Coffee/audio-visual-team to see what the default should be for Brownbags:

![image](https://user-images.githubusercontent.com/360261/120697213-718cb380-c47b-11eb-8117-64bdc9cda21e.png)

- Record active speaker with shared screen
- Record gallery view with shared screen
- Record active speaker, gallery view and shared screen separately
  - Active speaker
  - Gallery view
  - Shared screen 

Here's the Zoom article about the choices:

https://support.zoom.us/hc/en-us/articles/360025561091

**Comment by ClJarvis** [comment]
I think "Record active speaker with shared screen" would be best for viewing a replay. I don't understand why you would record them separate. Would you the have to stitch them back together? Have the shared screen most the time then add speaker for certain sections of the video. 
If the brownbag is showing code  I'd like to see that on screen as large as possible.

**Reply by danieltott** [reply]
Yeah - I think "Active speaker with shared screen" is the best default option. With all of these checkboxes checked, it produces a bunch of different video files (including the active speaker/screenshare combo), which gives you a lot of different options/control if you were going to do hand-editing etc. But - if they're never going to be used, it's easy enough to just turn them off too.

I go back and forth on this a lot - this is good feedback!

**Comment by bogdaaamn** [comment]
Heyyyy @danieltott thanks for bringing this up, I always think about it when going through the recording files 😂  (also surprised by how many options does Zoom provide).

We do use only "Active speaker with shared screen" at the moment, so I do believe is the best default option as well. It shows the active speaker if there is no screen shared, and it places the active speaker in the top right hand corner, and it shows the whole shared screen if there is one. So it is actually all we need and it's the perfect setup.

Although I really like to have options 😂 There is not really a frequent use case for the rest of them. There are possible use cases, but I don't think they will occur too often so we need them as default. _Eg_ use the gallery view if we record panels or conversation with multiple people, use separate feeds of active speaker and the screen if we want to do some montage shenanigans. Maybe we can enable them if we anticipate their usage?
